Aquifer/Proxy.swift

```diff
@@ -82,6 +49,51 @@ public struct Proxy<UO, UI, DI, DO, FR> {
     }
 }
 
+/// A `Proxy` is so named because it can represent many different kinds of information flows.  There
```

Comment:
Can this bit be moved to the docs for Proxy?  As is, it's placed kind of confusingly, and won't show up in Quick Help.